src/org/skypro/skyshop/basket/ProductBasket.java

```diff
@@ -0,0 +1,73 @@
+package org.skypro.skyshop.basket;
+
+import org.skypro.skyshop.product.Product;
+
+public class ProductBasket {
+    private static final Product[] products = new Product[5];
+
+    public void addProduct(Product product) {
+        boolean result = true;
+        for (int i = 0; i < products.length; i++) {
+            if (products[i] == null) {
+                products[i] = product;
+                result = true;
+                break;
+            } else {
+                result = false;
+            }
+        }
+        if (!result) {
+            System.out.println("Невозможно добваить продукт");
+            System.out.println();
+        }
+    }
+
+    public int printBasketCost() {
```

Comment:
```suggestion
    public int getBasketCost() {
```
print - это напечатать, метод НЕ печатает, а возвращает. Поэтому лучше начать с get